docs/ABOUT.md

```diff
@@ -91,19 +97,39 @@ This allows us to:
 3. Alter the files from the standard Drupal core installation without making
 changes to Drupal core's git.
 
-In our starter we use scaffolding to alter the files for two different
-scenarios.
+These files we alter are kept in `/assets` directory.
+They are modified using one of the following scenarios:
 
-1. Ignore files
-  * .htaccess (becuase we don't need it)
-  * development.services.yml (becuase we server our own)
+1. Skip files 
+  * .htaccess - because we don't need it
+  * development.services.yml - we use docksal to create our own version
+  * example.settings.local.php - we use comment out everything in lieu of our 
+  custom `settings.kanopi.php` file. We leave this just in case a developer 
+  needs to add settings on their local using the Drupal way.
 2. Append additional information to existing files.
   * robots.txt
-  * settings.php
-
-These files are kept in /assets and allow us to keep the original settings.php
-and robots.txt files in the default state, but add the customizations we need
-for our project.
+3. Replace assets
+  * settings.php - better inclusion of alternate settings files and settings
+  * settings.migration.php - for connecting to cloud-based source database
+
+### File Details
+
+**initial.settings.php**: this file is used to replace Drupal Core's `settings.php` file.
+It uses the same process that the `pantheon-systems/drupal-integrations` package uses but
+sets up the alternate settings files we use in our build cycle. We include the
+settings files in this order:
+
+1. pantheon - will exist on all environment
```

Comment:
1. Can you put in the full file name, ie `pantheon.settings.php`
2. "will exist on all environment" should be `environments`

Reply:
done